### PR TITLE
Introduce query for obtaining trove classfiers

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/thoth-application/issues/1850
Related: https://github.com/thoth-station/adviser/pull/2011

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -4724,6 +4724,48 @@ def get_solver_run_document_ids_all(
 
             return [obj[0] for obj in document_ids]
 
+    def get_python_trove_classifiers_all(
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:4727]: `get_python_package_version_trove_classifiers_all`?

**Review by pacospace — COMMENTED** [review]
One small comment :) otherwise lgtm! 🍺

File: thoth/storages/graph/postgres.py (hunk 1 above)
Comment (fridex) [reply thoth/storages/graph/postgres.py]: Renamed in a0bd153, PTAL.

**Review by pacospace — APPROVED** [review]
Thanks 💯

**Pull request merged**